_layouts/legacy-home.html

```diff
@@ -0,0 +1,21 @@
+{%- include doc-top.html -%}
+
+{%- include head.html -%}
+
+<body class="legacy-homepage">
```

Comment:
The purpose of this template is to allow `.legacy-homepage` specific hacks without making things messy elsewhere.